packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -157,8 +166,16 @@ const ScrollAppWrapper = ({ url, children }: ScrollAppWrapperProps) => {
     return <AppWrapper ref={ref}>{children}</AppWrapper>;
 };
 
-const BodyNormal = ({ url, menu, appMenu, children, guideOpen, isMenuInline }: NormalBodyProps) => (
-    <Body>
+const BodyNormal = ({
+    url,
+    menu,
+    appMenu,
+    children,
+    guideOpen,
+    isMenuInline,
+    isModalOpen,
+}: NormalBodyProps) => (
+    <Body elsOpen={isModalOpen && guideOpen}>
```

Comment:
what is `els`?

Reply:
elements

Reply:
please rename, for me it's also hard to guess what does it mean

Reply:
renamed to `isGuideAboveModal` a7b4d11